### DISCUSSION TITLE
对不起我是SB，这是我没添加样式搞出来的问题

### DISCUSSION THREAD

**Discussion by StarWEB890** [walinejs/waline#1264] [Q&A] [answered]
是这样的，我是[Hexo-Theme-SinGO](https://github.com/StarWEB890/hexo-theme-singo)主题的开发者
几天前我在SinGO主题使用Waline评论的时候手机访问出现了BUG
BUG如图：
![msedge_xqNXvBBXeV](https://cdn.staticaly.com/gh/StarWEB890/TuChuang@master/images/msedge_xqNXvBBXeV.4aby2c04ppi0.webp)
![msedge_2ceimzePFj](https://cdn.staticaly.com/gh/StarWEB890/TuChuang@master/images/msedge_2ceimzePFj.2vhzqknxifm0.webp)
使用其它评论正常（以Discuss、Twikoo作实例）：
![msedge_Q82ti0A8S2](https://cdn.staticaly.com/gh/StarWEB890/TuChuang@master/images/msedge_Q82ti0A8S2.396sb2lderu.webp)
![msedge_tFTmlokOL8](https://cdn.staticaly.com/gh/StarWEB890/TuChuang@master/images/msedge_tFTmlokOL8.6gq7x471jr00.webp)

**Comment by lizheming** [comment]
实在是没看明白，什么 BUG？

**Reply by StarWEB890** [reply]
我简略一下问题吧：
就是SinGO主题用Waline评论的时候整个页面右边会空出来一格
用其它评论不会

**Comment by Mister-Hope** [comment, marked as answer]
你自己写了什么样式还要别人给你找
# ?

还要美其名曰为 bug
# ?

你要觉得适配有问题或者不会是配你大可不必适配，不要过来丢两张图问我们为什么不work。

**Reply by StarWEB890** [reply]
对不起我是SB，我忘记定义.wl-emoji-popup的样式了

**Reply by Mister-Hope** [reply]
......

**Comment by StarWEB890** [comment]
对不起我是SB
![TIM_rBJYWcNlp1](https://cdn.staticaly.com/gh/StarWEB890/TuChuang@master/images/TIM_rBJYWcNlp1.61ae2c8j79k0.webp)
![TIM_ztK2vALML4](https://cdn.staticaly.com/gh/StarWEB890/TuChuang@master/images/TIM_ztK2vALML4.380grcjxpjc0.webp)
![msedge_TZSS1J4ZZZ](https://cdn.staticaly.com/gh/StarWEB890/TuChuang@master/images/msedge_TZSS1J4ZZZ.4ck3gceoeva0.webp)
